tm1637.py

```diff
@@ -199,6 +199,34 @@ def temperature(self, num):
             self.write(self.encode_string(string))
         self.write([_SEGMENTS[38], _SEGMENTS[12]], 2)  # degrees C
 
+    def dectemperature(self, num):
+        if num < -9.9: # limit to single digit negatives
+            self.write([0, 0, 0, 0])
+            self.show('lo')  # low
+        elif num > 99.9:
+            self.write([0, 0, 0, 0])
+            self.show('hi')  # high
+        else:
+            intval = int(num/1)
+            if num == 0: # exact zero
+                seg1 = 0b00000000
+                seg2 = self.encode_digit(0)
+                seg3 = self.encode_digit(0)
+            else:
+                if abs(intval) < 10 and num > 0: # single digit positive
+                    seg1 = 0b00000000
+                    seg2 = self.encode_digit(abs(intval))
+                else: # two digit (can only be positive)
+                    seg1 = self.encode_digit(int(abs(intval)/10))
+                    seg2 = self.encode_digit(int(num-(int(abs(intval)/10)*10)))
+                if num < 0: # negative
```

Comment:
This line can be an "elif" for the "if" above

Reply:
Thanks for the feedback. I understand and followed the other suggestions. Here I can't quite follow you.